tajo-core/src/main/java/org/apache/tajo/engine/planner/global/GlobalPlanner.java

```diff
@@ -163,7 +164,7 @@ public void build(QueryContext queryContext, MasterPlan masterPlan) throws IOExc
   private static void setFinalOutputChannel(DataChannel outputChannel, Schema outputSchema) {
     outputChannel.setShuffleType(NONE_SHUFFLE);
     outputChannel.setShuffleOutputNum(1);
-    outputChannel.setStoreType("TEXT");
+    outputChannel.setStoreType(BuiltinStorages.DRAW);
```

Comment:
How about adding a final variable for the default store type and using it in the class?